packages/core/src/routes/user.ts

```diff
@@ -36,13 +37,24 @@ export default function userRoutes<T extends AnonymousRouter>(router: T) {
         passwordEncryptionMethod
       );
 
-      ctx.body = await insertUser({
+      await insertUser({
```

Comment:
wondering if any db insert error need to be handled?

Reply:
see `packages/core/src/middleware/koa-error-handler.ts`